doc/source/serve/tutorials/hpu_inference_serve.md

```diff
@@ -0,0 +1,291 @@
+| N/A   25C   N/A   112W / 600W |    768MiB / 98304MiB |     0%           N/A |
+|-------------------------------+----------------------+----------------------+
+|   5  HL-225              N/A  | 0000:0d:00.0     N/A |                   0  |
+| N/A   26C   N/A   111W / 600W |  26835MiB / 98304MiB |     0%           N/A |
+|-------------------------------+----------------------+----------------------+
+|   6  HL-225              N/A  | 0000:0f:00.0     N/A |                   0  |
+| N/A   24C   N/A    93W / 600W |    768MiB / 98304MiB |     0%           N/A |
+|-------------------------------+----------------------+----------------------+
+|   7  HL-225              N/A  | 0000:0e:00.0     N/A |                   0  |
+| N/A   25C   N/A    86W / 600W |    768MiB / 98304MiB |     0%           N/A |
+|-------------------------------+----------------------+----------------------+
+| Compute Processes:                                               AIP Memory |
+|  AIP       PID   Type   Process name                             Usage      |
+|=============================================================================|
+|   0        N/A   N/A    N/A                                      N/A        |
+|   1        N/A   N/A    N/A                                      N/A        |
+|   2        N/A   N/A    N/A                                      N/A        |
+|   3        N/A   N/A    N/A                                      N/A        |
+|   4        N/A   N/A    N/A                                      N/A        |
+|   5        N/A   N/A    N/A                                      N/A        |
+|   6        N/A   N/A    N/A                                      N/A        |
+|   7        N/A   N/A    N/A                                      N/A        |
++=============================================================================+
+```
+
+If it works, you can then start a container like this:
+```bash
+docker pull vault.habana.ai/gaudi-docker/1.14.0/ubuntu22.04/habanalabs/pytorch-installer-2.1.1:latest
+docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none --cap-add=sys_nice --net=host --ipc=host vault.habana.ai/gaudi-docker/1.14.0/ubuntu22.04/habanalabs/pytorch-installer-2.1.1:latest
+```
+
+Note that you may want to mount the directory containing the examples into the container, as well as the models.
+
+Inside the container, run
+```bash
+pip install ray[tune,serve]
+pip install git+https://github.com/huggingface/optimum-habana.git
+# 1.14.0 should be replaced with the driver version of the container
+pip install git+https://github.com/HabanaAI/DeepSpeed.git@1.14.0
+# Only needed by the deepspeed example
+export RAY_EXPERIMENTAL_NOSET_HABANA_VISIBLE_MODULES=1
+```
+
+Start a ray cluster via `ray start --head` and you are ready to run the examples.
+
+## Single HPU Example
+
+This example shows how to deploy a Llama2-7b model on Gaudi for inference. 
```

Comment:
Try to use one term (HPU/Gaudi) consistently; otherwise, it get a bit confusing what the tutorial is referring to.

```suggestion
This example shows how to deploy a Llama2-7b model on an HPU for inference. 
```